Packages/com.unity.render-pipelines.high-definition/Runtime/Compositor/CompositionLayer.cs

```diff
@@ -228,7 +228,7 @@ public int pixelHeight
         }
         public void Init(string layerID = "")
         {
-            if (m_LayerName == "")
+            if (m_LayerName?.Length == 0)
```

Comment:
I automatically add this for new code I write.
According to the benchmarks I did this did not add any runtime performance, but without it might be marginally faster and closer to the original code